### PR TITLE
Minor fixes for unix build infra.

### PR DESCRIPTION
**Pull request by codito**
Add tools directory to gitignore. Use NuGet.config case sensitive for package resolution.

### CODE REVIEW

**Comment by harshjain2** [comment]
@dotnet-bot test this please

File: .gitignore [hunk 1]
```diff
@@ -124,4 +124,4 @@ src/package/sign/sign.nuget.targets
 *.ru.resx
 *.tr.resx
 *.zh-Hans.resx
-*.zh-Hant.resx
\ No newline at end of file
+*.zh-Hant.resx
```
Comment (singhsarab) [inline .gitignore]: Didn't understand this change.

**Review by singhsarab — APPROVED** [review]
👍

File: .gitignore (hunk 1 above)
Comment (codito) [inline .gitignore]: There wasn't a newline at end of file earlier.

**Pull request merged**